content/en/case-studies/cricket-analytics.md

```diff
@@ -0,0 +1,159 @@
+           attr="*(Image Credits: connect.vin)*"
+           attrlink="https://connect.vin/2019/05/ai-for-cricket-batsman-pose-analysis/" >}}
+
+### The Challenges
+
+* **Data Cleaning and preprocessing**
+
+  IPL has expanded cricket beyond the classic test match format to a much
+  larger scale. The number of matches played every season across various
+  formats has increased and so has the data, the algorithms, newer sports data
+  analysis technologies and simulation models.  Cricket data analysis requires
+  field mapping, player tracking, ball tracking, player shot analysis and
+  several other aspects involved in how the ball is delivered, its angle, spin,
+  velocity and trajectory. All these factors together have increased the
+  complexity of data cleaning and preprocessing.
+
+* **Dynamic Modeling**
+
+  In cricket, just like any other sport,
+  there can be a large number of variables related to tracking various numbers
+  of players on the field, their attributes, the ball and several possibilities
+  of potential actions.  The complexity of data analytics and modeling is
+  directly proportional to the kind of predictive questions that are put forth
+  during analysis and are highly dependent on data representation and the
+  model.  Things get even more challenging in terms of computation, data
+  comparisons when dynamic cricket play predictions are sought such as what
+  would have happened if the batsman had hit the ball at a different angle or
+  velocity.
+
+* **Predictive Analytics Complexity**
+
+  Much of the decision making in Cricket is based on questions such as "how
+  often does a batsman play a certain kind of shot if the ball delivery is of a
+  particular type", or "how does a bowler change his line and length if the
+  batsman responds to his delivery in a certain way".
+  This kind of predictive analytics query requires highly granular dataset
+  availability and the capability to synthesize data and create generative
+  models that are highly accurate.
+
+## NumPy’s Role in Cricket Analytics
+
+Sports Analytics is a thriving field. Many researchers and companies
+[use NumPy](https://adtmag.com/blogs/dev-watch/2017/07/sports-analytics.aspx)
+and other PyData packages like Scikit-learn, SciPy, Matplotlib, and Jupyter.
+in addition to latest machine learning and AI techniques.  NumPy has been used
+for various kinds of cricket related sporting analytics such as:
+
+* **Data Visualization:** Data graphing and [visualization](https://towardsdatascience.com/advanced-sports-visualization-with-pandas-matplotlib-and-seaborn-9c16df80a81b)
+  provides useful insights into relationship between various datasets;
+  [causal analysis](https://amplitude.com/blog/2017/01/19/causation-correlation),
+  and [big data approaches](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4996805/)
+  are used for tactical analysis. NumPy offers the core numeric computing foundation for several
+  such analyses.
```

Comment:
This is the last section that I think needs a little reshuffle. The second part of the paragraph above talks about causal analysis, which belongs in `Statistical Analysis` below instead. The `big data approaches` fits better there as well.

Reply:
I made this change